src/main/java/school/lemon/changerequest/java/introduction/hw1/Task3.java

```diff
@@ -2,6 +2,6 @@
 
 public class Task3 {
     public static boolean isBetween(int a, int b, int c) {
-        return a <= b && c >= b || c <= b && b >= a || a > b && c < b;
+        return b>=a && b<=c || b>=c && b>=a || b<a && b>c;
```

Comment:
You didn't get what I mean.
The only valuable expression here will be:
`return b>=a && b<=c || b>=c && b<=a;`

If you follow your second term, you will not satisfy task requirements, as you should return `true` only when `b` is between `a` and `c`.
In you second term `b` is greater or equal to `c` and `b` is greater or equal to `a`. From this term only one part is correct regarding task requirements: `b==c && b==a`.